### ISSUE TITLE
Issue with right-click on any tab

### ISSUE THREAD

**Issue by iRay** [romaindalichamp/TerminalTabTailor#9]
If we make a right-click on any tab an exception appears.
Exception is raised only at the first plugin invoke.
Intellij Idea Ultimate Edition: Build #IU-233.14475.28, built on February 13, 2024

```
com.intellij.diagnostic.PluginException: Empty menu item text for CustomRenameTerminalSessionAction@ToolwindowPopup (com.terminaltabtailor.actions.CustomRenameTerminalSessionAction). The default action text must be specified in plugin.xml or its class constructor [Plugin: com.terminaltabtailor.TerminalTabTailor]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:89)
	at com.intellij.openapi.actionSystem.impl.Utils.reportEmptyTextMenuItem(Utils.kt:597)
	at com.intellij.openapi.actionSystem.impl.Utils.filterInvisible(Utils.kt:564)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenuInner(Utils.kt:468)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu$intellij_platform_ide_impl(Utils.kt:401)
	at com.intellij.openapi.actionSystem.impl.Utils.fillPopupMenu(Utils.kt:367)
	at com.intellij.openapi.actionSystem.impl.ActionPopupMenuImpl$MyMenu.updateChildren(ActionPopupMenuImpl.java:174)
	at com.intellij.openapi.actionSystem.impl.ActionPopupMenuImpl$MyMenu.show(ActionPopupMenuImpl.java:124)
	at com.intellij.openapi.wm.impl.content.ToolWindowContentUi.showContextMenu(ToolWindowContentUi.java:638)
	at com.intellij.openapi.wm.impl.content.ToolWindowContentUi$4.invokePopup(ToolWindowContentUi.java:574)
	at com.intellij.ui.PopupHandler.mousePressed(PopupHandler.java:47)
```

**Comment by romaindalichamp** [comment]
Hey iRay,

Thank you ! i also got this issue while working on another feature. I am working on it right now, it will very probably be part of 1.2.0

**Comment by romaindalichamp** [comment]
Hey iRay,

Thanks again for your feedback regarding this issue, it should be solved in the version 1.2.0 which is currently undergoing JetBrains' review process and is expected to be available on the marketplace within a few days.

[edit]: now available :)

if you want the new version including the fix before it's official release, you can download the version 1.2.0 as a Jar here: https://plugins.jetbrains.com/plugin/23903-terminal-tab-tailor/versions/in%20review

To install, you can copy/paste the downloaded Jar to the Plugins directory following the instructions here (and remove the previous version): https://www.jetbrains.com/help/idea/directories-used-by-the-ide-to-store-settings-caches-plugins-and-logs.html#plugins-directory

Let me know how it works for you.

Romain
